swift/ql/test/library-tests/dataflow/flowsources/CONSISTENCY/CfgConsistency.expected

```diff
@@ -0,0 +1,5 @@
+deadEnd
+| alamofire.swift:469:9:469:9 | stream |
+| alamofire.swift:484:9:484:9 | stream |
+| alamofire.swift:499:9:499:9 | stream |
+| alamofire.swift:514:9:514:9 | stream |
```

Comment:
How concerned should we be about these?

Reply:
These are not the only ones we have, so I'm not sure if it gets much worse with these changes.

Reply:
... I see that you've said elsewhere they should be addressed separately.  That's fine, as long as there's an issue and any breakage before then is limited.

Reply:
> These are not the only ones we have, so I'm not sure if it gets much worse with these changes.

Fair enough.  My intuition is there's a decent chance these kinds of problems do cost us data flow and thus results, so they should not be ignored in the long run.

Reply:
I filed an issue. I think the fix should be trivial as we need to add some wires for the new AST node in CFG.

Reply:
And attempted to fix it here https://github.com/github/codeql/pull/14717

Reply:
🎉